### PR TITLE
chore(standoff): Use Base64-encoded UUIDs in standoff tags

### PR DESCRIPTION
**Pull request by benjamingeer**
This PR stores the UUIDs of standoff tags in Base64 encoding to make them shorter. This should make triplestore query results somewhat shorter. The change is backwards-compatible; existing UUIDs will still work.

- [x] Change `StandoffTagV2` to store its UUID as a `java.util.UUID` object instead of a string.
- [x] Change SPARQL update templates to use `KnoraIdUtil` to Base64-encode standoff UUIDs.
- [x] Change `StandoffTagUtilV2` to decode Base64-encoded UUIDs.

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
@benjamingeer I will do the review on Monday, ok?

**Comment by benjamingeer** [comment]
> I will do the review on Monday, ok?

No problem, thanks!

**Comment by tobiasschweizer** [comment]
On Monday, I will have fresh energy and be extra annoying ;-)

**Comment by benjamingeer** [comment]
Ich freue mich auf deine Erbsenzählerei. :)

File: webapi/src/main/scala/org/knora/webapi/util/standoff/StandoffTagUtilV2.scala [hunk 1]
```diff
@@ -774,7 +778,7 @@ object StandoffTagUtilV2 {
                         case Some(endIndex: String) => Some(endIndex.toInt)
                         case None => None
                     },
-                    uuid = standoffNodes(OntologyConstants.KnoraBase.StandoffTagHasUUID),
+                    uuid = knoraIdUtil.decodeUuid(standoffNodes(OntologyConstants.KnoraBase.StandoffTagHasUUID)),
```
[the remark this answers is not in the document]
Comment (benjamingeer) [reply webapi/src/main/scala/org/knora/webapi/util/standoff/StandoffTagUtilV2.scala]: Yes, that’s right.

**Comment by benjamingeer** [comment]
Thanks for the review!

**Pull request merged**